Dockerfile.bundle

```diff
@@ -0,0 +1,36 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.15 as build
+
+ENV GO111MODULE=on
+ENV GOFLAGS=""
+
+RUN yum install -y python3-pip && \
+    yum clean all
+RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | \
+    bash -s -- 3.8.8 && mv kustomize /usr/bin/
+RUN pip3 install waiting==1.4.1 mkdocs==1.1.2
+RUN curl -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.6.0/openshift-client-linux-4.6.0.tar.gz | tar -xz -C /usr/local/bin
```

Comment:
May as well get the 4.7 client.

Reply:
Additionally, I think it may be better to simply have a: `COPY --from=registry.svc.ci.openshift.org/ocp/4.7:cli /usr/bin/oc`

Reply:
Great tip. Didn't know COPY could do that. For local development I haven't found a way to use registry.svc.ci.openshift.org without having to authenticate. Would using quay.io/openshift/origin-client:4.7 be ok?

Reply:
I think `quay.io/openshift/origin-client` would be alright.